pkg/services/pluginsintegration/pluginstore/plugins.go

```diff
@@ -32,7 +33,9 @@ type Plugin struct {
 	AngularDetected bool
 
 	// This will be moved to plugin.json when we have general support in gcom
-	Alias string `json:"alias,omitempty"`
+	Alias string
+
+	ExternalService *oauth.ExternalService
```

Comment:
This is unused as of right now (always nil) or am I missing something?
I think you need to add here:

https://github.com/grafana/grafana/blob/f73a8911096bc6913a81713f52127c23f2b275e6/pkg/services/pluginsintegration/pluginstore/plugins.go#L79

```go
diff --git a/pkg/services/pluginsintegration/pluginstore/plugins.go b/pkg/services/pluginsintegration/pluginstore/plugins.go
index 4d817042e8a..4617f48b377 100644
--- a/pkg/services/pluginsintegration/pluginstore/plugins.go
+++ b/pkg/services/pluginsintegration/pluginstore/plugins.go
@@ -77,5 +77,6 @@ func ToGrafanaDTO(p *plugins.Plugin) Plugin {
                BaseURL:           p.BaseURL,
                AngularDetected:   p.AngularDetected,
                Alias:             p.Alias,
+               ExternalService:   p.ExternalService,
        }
 }
```

Reply:
You're bang on. Fixed 👍